### ISSUE TITLE
Unable to Jailbreak iPhone 5s

### ISSUE THREAD

**Issue by schizodeviant** [checkra1n/BugTracker#2264]
**Tell us about your setup:**
1. What iDevice are you using? iPhone 5S
2. On what version of iOS is it? 12.5.5
3. What version of checkra1n are you using? 0.12.4 beta
4. What is your host system (OS version? Hackintosh? VM? etc.)? Windows 8
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)? USB-A

**What are the steps to reproduce the issue?**
1.  alt-f2, then start, start, enter into dfu mode

...

**What do you expect, and what is happening instead?**
done jailbreaking -> Exploit Failed (Error Code: 31) Info: Task Checkrain: 657 blocked for more than 120 seconds

**Did you see a popup on the device stating it entered "Safe Mode" due to an error?**
no

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**
Yes, but with different message:  Exploit Failed (Error Code: 31) Info: Task Checkrain: 673 blocked for more than 120 seconds

**Any other info, error logs, screenshots, ...?**
Steps I tried: Restarted phone and PC, and [x] boxes which says safe mode and allowing untested versions
![PXL_20220321_150532863](https://user-images.githubusercontent.com/73815747/159217708-f4cb268d-45ef-4ccc-87b0-3b1b5371897f.jpg)

**Comment by mrbiggleswirth** [comment]
checkra1n doesn't work on Windows.
https://www.theiphonewiki.com/wiki/Checkra1n

https://checkra.in/releases/
checkra1n 0.12.4 beta
> Unsupported platforms
> - This beta is only available for macOS and Linux. Work is ongoing to support Windows, which will be added in a later release.

> Known issues
> - A7 devices (iPhone 5s) will currently not work with the Linux version

**Comment by 0E800** [comment]
Did you even look at the screenshot? Its not windows.

I also have same issue with iPhone 5s. I would include screenshot but its the same as OP.

Its not tainted.

**Comment by mrbiggleswirth** [comment]
@0E800 Yes, I looked at his screenshot the monitor said HP and he wrote this in his specification:

> Tell us about your setup:
> 4. What is your host system (OS version? Hackintosh? VM? etc.)? Windows 8

None of those pieces of information looks like he is using a **macOS computer** to jailbreak his iPhone with checkra1n. That is why it fails according to my experience, jailbreaking with most versions of checkra1n program.

**Comment by 0E800** [comment]
Apologies for late reply.
The livecd or Bootable USB Linux checkra1n environment is what I was using. You suggest that using a native Apple branded device will produce positive results?

**Comment by mrbiggleswirth** [comment]
@0E800 Yes using Apple only hardware and software is the path of least resistance when it comes to Jailbreaking in general.

Especially for checkra1n, their documentation and support for Linux never seemed to be up to date for the last 6 months.
